assignments/lambda-classes.js

```diff
@@ -1 +1,201 @@
 // CODE here for your Lambda Classes
+class Person{
+    constructor(personAttributes){
+    this.name = personAttributes.name;
+    this.age = personAttributes.age;
+    this.location = personAttributes.location;
+    }
+    speak(){
+        console.log(`Hello my name is ${this.name}, I am from ${this.location}.`)
+    }
+}
+
+class Instructor extends Person{
+    constructor(instructorInfo){
+        super(instructorInfo);
+        
+        this.specialty = instructorInfo.specialty;
+        this.favLanguage = instructorInfo.favLanguage;
+        this.catchPhrase = instructorInfo.catchPhrase;
+     }
+  
+     demo(subject){
+    console.log(`Today we learned about ${subject}.`)
+    }
+
+    grade(student, subject){
+        console.log(`${student.name} receives a perfect score on ${subject}`);
+    }
+    points(student){
+        student.grade = student.grade + ((Math.random() * 20) - 10);
```

Comment:
This will make this method work. 
points(student){
        return (Math.random() * 100);
 }
I think with more time you would have gotten your own resolution. Math.rand chooses a number between 0-.999999.